docs/analyzers/Using Additional Files.md

```diff
@@ -0,0 +1,231 @@
+Introduction
+============
+
+This document covers the following:
+
+* Uses of Additional Files
+* Passing Additional Files on the command line
+* Specifying an individual item as an `AdditionalFile` in an MSBuild project
+* Specifying an entire set of items as `AdditionalFile`s in an MSBuild project
+* Accessing and reading additional files through the `AnalyzerOptions` type
+* Includes sample analyzers that read additional files
+
+Uses
+====
+
+Sometimes an analyzer needs access to information that is not available through normal compiler inputs--source files, references, and options. To support these scenarios the C# and Visual Basic compilers can accept additional, non-source text files as inputs.
```

Comment:
It may be good to give an example here where additional file would be needed by the analyzer.